languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -8325,6 +8325,10 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
                 <token regexp="yes">&prefixes;</token>
                 <token postag="[NJ] . (p|sp)|V.* ppa . (p|sp)" postag_regexp="yes"/>
             </antipattern>
+            <antipattern>
+                <token postag="Y"/>
+                <token regexp="yes">Vendémiaire|Brumaire|Frimaire|Nivôse|Pluviôse|Ventôse|Germinal|Floréal|Prairial|Messidor|Thermidor|Fructidor</token>
```

Comment:
question:will that also cover the form without uppercase (which is suppose to be used)?

Reply:
Yep, it worked in the main test :)